examples/opentelemetry/envoy-1.yaml

```diff
@@ -25,6 +25,16 @@ static_resources:
                     cluster_name: opentelemetry_collector
                   timeout: 0.250s
                 service_name: envoy-1
+          access_log:
```

Comment:
i think these `access_log` changes are the real change in the PR

im not totally clear what the purpose is but if we are going to change these then the docs and `verify.sh` script should reflect that

Reply:
> i think these access_log changes are the real change in the PR

Pretty much, but also updating the collector to a more recent version and adjusting the configs is worthy in its own right.

Reply:
unless i misunderstand tho - the version isnt updated - just made explicit

happy to carry some cleanups, but i think the change is probably great and requires focus

to land, we really need to update the docs to explain, and relatedly to test what the docs say to prevent bitrot etc

Reply:
You are right, the version did not actually change. Before updating the docs, I would love for the correlation to... well actually work (see #30268). My idea to open the PR this early was to give @ashishb-solo a quick way to test the changes, and to motivate why the PR to actually fix the correlation was needed.

Reply:
cool, thanks for context

is it the same issue as https://github.com/envoyproxy/envoy/issues/33906 ?

i reopened the one you posted but i have a feeling that they are now duping

with these sandboxes they are documentation first, so to justify adding a load of configuration it needs to be documented and tested

Reply:
@phlax : #33906 is about the correlation not working. And it still doesn't, it's broken at a technical level. This demo will start working as expected, and showcase the correlation, when the underlying bug in the `opentelemetry` access log exporter is fixed :-)

Reply:
FYI #33909 solves the problem pretty nicely, see https://github.com/envoyproxy/envoy/pull/33909#issuecomment-2092412409